### PR TITLE
created: hashtag input and button input

### PR DESCRIPTION
**Pull request by CorentinReims**
Forget to create an input branch

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,12 +1,17 @@
 import React, { Component } from "react";
 import "./App.css";
+import "./Hashtaginput.css"
```
Comment (dadtmt) [inline src/App.js]: this should be import in HashtagInput.js

File: src/Hashtaginput.css [hunk 2]
```diff
@@ -0,0 +1,3 @@
+.sizeinput {
```
Comment (dadtmt) [inline src/Hashtaginput.css]: name this file with CamelCase

File: src/buttoninput.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import React, { Component } from 'react'
```
Comment (dadtmt) [inline src/buttoninput.js]: no need of this component

**Review by dadtmt — CHANGES_REQUESTED** [review]

**Comment by CorentinReims** [comment]
New commit resolving mistakes

**Review by dadtmt — APPROVED** [review]

**Pull request merged**